affinity/cpp-20/dXXX1r0.md

```diff
@@ -195,58 +266,147 @@ Below *(Listing 2)* is an example of executing a parallel task over 8 threads us
 ```
 *Listing 2: Example of using the bulk_execution_affinity property*
 
+### Proposed Wording
 
-## Header `<execution>` synopsis
+The `bulk_execution_affinity_t` property is a behavioral property as defined in P0443 [[22]][p0443] which describes the guarantees an executor provides to the binding of *execution agents* created by a call to `bulk_execute` to the underlying *threads of execution* and to the locality of those *threads of execution*.
+
+The `bulk_execution_affinity_t` property provides nested property types and objects as described below, where:
+* `e` denotes an executor object of type `E`,
+* `f` denotes a function object of type `F&&`,
+* `s` denotes a shape object of type `execution::executor_shape<E>`,
```

Comment:
need "and" after the comma, for the third point only

Reply:
Fixed.